islands/MiniMeDoll/MiniMe.tsx

```diff
@@ -38,9 +41,29 @@ interface FinishStep {
 }
 
 export default function MiniMe(props: Props) {
+  console.log("NewdollParts: ", props.dollParts.parts)
   
   const [IsOpen, setIsOpen] = useState(false)
 
+  const step = useSignal(Number(localStorage.getItem("step") || "1"));
+  
+  const changeStep = (operation: string) => {
```

Comment:
Padronize as operações
```suggestion
  const changeStep = (operation: 'increment' | 'decrement') => {
```